src/components/table-of-contents/_macro.njk

```diff
@@ -41,5 +42,44 @@
                 }}
             {% endif %}
         </nav>
+
+        {% if params.relatedLinks %}
+            <nav
+                class="ons-table-of-contents__related-links ons-u-pb-s ons-u-pt-l ons-u-mt-s"
+                aria-label="{{ params.relatedLinks.ariaLabel | default('Related links') }}"
+            >
+                <h2 class="ons-table-of-contents__title ons-u-fs-r--b ons-u-mb-s">{{ params.relatedLinks.title }}</h2>
+
+                {{
+                    onsList({
+                        "variants": 'bare',
```

Comment:
```suggestion
                        "variants": 'dashed',
```
I think this needs to use the dashed variant looking at this image on the card

<img width="271" alt="Screenshot 2025-04-07 at 09 57 46" src="https://github.com/user-attachments/assets/4397ee4b-7646-4919-a171-2945f63f8f13" />

Reply:
Ah we agreed with Joe to deviate from the designs and remove the dashes for the related links, to make the distinction between the types of links (dashed = same page/anchor link and undashed = link to another page). There's a note of this in the ticket.